features/single-sign-on/README.md

```diff
@@ -44,6 +48,12 @@ Alternatively, they may use the SSO application on your IdP portal to initiate a
 
 Upon successful authentication response from the IdP, ImageKit will check if there have been any changes to the user's information (such as full name and ImageKit role) since their last login on ImageKit. If so, the information will be updated on ImageKit automatically. 
 
+### SSO and MFA
```

Comment:
Let's expand the MFA heading here.

I think it would be better to keep all related information under this section only instead of breaking the information into the hint above and then under this section.

Please run all the text through grammarly or quillbot.

Reply:
expanded the MFA. Ran the text through quillbot. We can remove the hint above. @nikniv Thoughts?

Reply:
Not necessary to remove the hint. Might as well highlight it for users